lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -222,7 +222,7 @@ def queue_job(self, job_wrapper):
         log.info("(%s) queued as %s" % (galaxy_id_tag, external_job_id))
 
         # store runner information for tracking if Galaxy restarts
-        job_wrapper.set_job_destination(job_destination, external_job_id)
+        job_wrapper.set_external_id(external_job_id)
```

Comment:
Is there a reason not to do the same for all job runners ?

Reply:
Probably it is worthwhile to review the other job runners and check if any of them modify the job destination - I'm just conservative and wasn't focused on that path. Feel free to apply the change to the rest of them.

Reply:
ok, will do that!